packages/api/src/beacon/routes/events.ts

```diff
@@ -289,7 +288,6 @@ export function getTypeByEvent(config: ChainForkConfig): {[K in EventType]: Type
   };
 }
 
-// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
so return types don't need to be explicit anymore?

Reply:
Currently for that rule, biomejs didn't differentiate between functions and anonymous functions. So we get over a thousand errors in our codebase for anonymous functions. I hope they will extend this rule in upcoming release and will  bring it to error level. 

https://biomejs.dev/linter/rules/use-explicit-function-return-type

Reply:
this one is sad, hoping this gets resolved quickly